### ISSUE TITLE
Är det ett Javascript-lib eller ett HTML5+JS-lib?

### ISSUE THREAD

**Issue by andersarpi** [Hanifb/marginal-calc#4]
> Installation:
> Du behöver endast inkludera marginalCalc.js samt ha en mapp som heter modules där alla moduler ligger

Detta nämner inte beroenden på index.html, Bootstrap, jQuery osv. Som det är nu beskrivs marginalCalc som "Marginaleffektskalkylator i HTML5 och Javscript".

Antingen får domänlogiken för att räkna ut datan separeras till ett eget lib, och demoimplementationen ett eget, eller så får beskrivningen av nuvarande repo uppdateras och beroenden på HTML, Bootstrap, jQuery osv inkluderas som en del av installationen.

Index.html ligger i ~/ men är beroende av filer som ligger i ~/demo/, vilket inte heller är tydligt och antagligen är ett symtom på samma sak.

Min rekommendation är att inte låtsas att "demo"-mappen bara är ett demo utan istället omfamna det faktum att det är innehållet i demo-mappen _i kombination med_ domänlogiken som är det intressanta och bör ses som den färdiga produkten.

**Comment by Hanifb** [comment]
Ja, det är frågan. Demot kan ju förlita sig på bootstrap och jquery från CDN. De två bilderna går att base64:a in i html filen.

Själva kalkylatorn är ju inte beroende av något av dem.

Vad tycker du? @holmstrom

**Comment by andersarpi** [comment]
Med puristens glasögon så borde JS-libbet vara helt separat från frontendimplementationen/exemplet. Det skulle ju gå att ha det i samma repo fortfarande, men kanske separera dem lite tydligare. Det är mer otydligheten i att index.html och marginalCalc.js ligger bredvid varandra som jag reagerar på.

Det skulle kanske vara lite tydligare om det såg ut något i stil med:

```
src/marginalCalc/marginalCalc.js
src/marginalCalc/modules/Bostadsbidrag.js
src/marginalCalc/modules/Tax.js (etc)
example/index.html
example/bootstrap/...
example/img/..
example/lib/jquery.js
example/lib/marginalcalc.js <- obs! refa in libet här igen och referera till det från index.html
example/lib/modules/Tax.js (etc) <- obs!
docs/... etc
```

Då blir det en tydlig uppdelning: här är dokumentation, här är själva biblioteket, här är ett exempel på hur man använder sig av biblioteket för att göra en interaktiv sajt.

**Comment by perlun** [comment]
I agree with @holmstrom, this makes sense.
